### PR TITLE
Freewebnovel: Regex to remove website spam

### PR DESCRIPTION
**Pull request by Palloxin**
It removes just a part of the spam.
EDIT: I tested that script with 120+ chapters on chapters published 1+ year ago. I didnt find any more spam.
I also checked recent chapters; The script still deletes that spam effectively.

### CODE REVIEW

File: src/plugins/english/freewebnovel.ts [hunk 1]
```diff
@@ -115,7 +115,8 @@ class FreeWebNovel implements Plugin.PluginBase {
       loadedCheerio('div.txt').find('p:last-child').remove();
 
     const chapterText = loadedCheerio('div.txt').html() || '';
-    return chapterText;
+    return chapterText
+      .replace(/[ƒf][rɾг][ēeё][ēёe][wω][ёēe][bɓ][nɳη][ø૦ѳσo][ѵv][ёeē][lℓɭ]\.\s?[ƈcç][૦σøoѳ][๓ɱm]\.?/gi, '');
```
Comment (nyagami) [inline src/plugins/english/freewebnovel.ts:119]: Does this kind of string have a specific tag or classname?

Comment (Palloxin) [reply src/plugins/english/freewebnovel.ts:119]: I am not sure. The website first loads the spam then removes it. Maybe to tackle web scrapers.

Comment (K1ngfish3r) [reply src/plugins/english/freewebnovel.ts:119]: > Does this kind of string have a specific tag or classname?

There is no tag/class name, it is baked directly into the <p> tag

**Review by nyagami — APPROVED** [review]

**Comment by Palloxin** [comment]
I think it's ready now

**Pull request merged**